src/index.js

```diff
@@ -0,0 +1,60 @@
+const listContainer = document.querySelector('.todolist-tasks-list');
+
+const lists = [{
+  index: 1,
+  completed: [true, false],
```

Comment:
- [ ] The the value of the `completed` key should be a boolean, and not an array of booleans, kindly change this 😄

Reply:
This issue is well resolved! ✔️